src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -26,17 +27,15 @@ type RampsListProps = {
 
 export const RampsList: React.FC<RampsListProps> = ({ setFiatRampProvider }) => {
   const history = useHistory()
-  const mtPelerinFiatRampFlag = useFeatureFlag('MtPelerinFiatRamp')
   const tagColor = useColorModeValue('gray.600', 'gray.400')
+  const featureFlags = useAppSelector(selectFeatureFlags)
   const ramps = useMemo(() => {
     type Entry = [keyof typeof supportedFiatRamps, SupportedFiatRampConfig]
     const initial: ReactElement[] = []
     const result = (Object.entries(supportedFiatRamps) as Entry[]).reduce(
       (acc, supportedFiatRamp) => {
         const [fiatRamp, fiatRampConfig] = supportedFiatRamp
-        // TODO: remove before merging to develop
-        if (fiatRamp === 'MtPelerin' && !mtPelerinFiatRampFlag) return acc
```

Comment:
i like your new generic implementation, can you please ensure mt pelerin is compatible with this?

Reply:
done